lib/get-config.js

```diff
@@ -307,20 +311,32 @@ function validateRules(rules, loadedRules, options) {
 
 function validateOverrides(config, options) {
   if (config.overrides) {
-    for (const override of config.overrides) {
+    for (let overrideConfig of config.overrides) {
       // If there are keys in the object which are not yet supported by overrides functionality, throw an error.
-      const overrideKeys = Object.keys(override);
-      const containsValidKeys = overrideKeys.every(item => SUPPORTED_OVERRIDE_KEYS.includes(item));
+      let overrideKeys = Object.keys(overrideConfig);
+      let containsValidKeys = overrideKeys.every(item => SUPPORTED_OVERRIDE_KEYS.includes(item));
       if (!containsValidKeys) {
         throw new Error(
           'Using `overrides` in `.template-lintrc.js` only supports `files` and `rules` sections. Please update your configuration.'
         );
       }
-      validateRules(override.rules, config.loadedRules, options);
+      validateRules(overrideConfig.rules, config.loadedRules, options);
+      overrideConfig = processRules(overrideConfig);
```

Comment:
I don't understand what is happening here. Why are we reassigning the variable (nothing uses it again)? Do we expect `processRules` to just mutate?